### DISCUSSION TITLE
Show the current buffer name?

### DISCUSSION THREAD

**Discussion by towry** [mrjones2014/legendary.nvim#193] [Ideas]
Occasionally after I select a command to execute but no effect being taken, after that I realize the active buffer is not the correct one.

For example, when I filter `NvimTreeFindFile` and press enter, no effect because the active buffer is the nvim-tree buffer, this happens a lot.

**Comment by mrjones2014** [comment]
Does this work for now?

```lua
require('legendary').find({ select_prompt = vim.api.nvim_buf_get_name(0) })
```

I can include more customization options/mechanisms in V2, see #191 coming soon

**Reply by towry** [reply]
Yes, this api does work as what I wanted, awesome!

**Reply by mrjones2014** [reply]
You can also set it as default via the .setup() function, it’s in the README

**Reply by towry** [reply]
Hi, there maybe is a minor typo in the README:

```
The require('legend').find() function takes an opts table with the following fields (all optional):
```

I think it should be `require('legendary')`

**Reply by mrjones2014** [reply]
You’re right. It will be fixed soon.
